doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -229,6 +229,9 @@ Serial LTE modem
   * The initialization of the application to ignore a failure in nRF Cloud module initialization.
     This occurs sometimes especially during development.
   * The initialization of the application to send "INIT ERROR" over to UART and show clear error log to indicate that the application is not operational in case of failing initialization.
+  * The ``AT#XTCPCLI`` and ``AT#XUDPCLI`` commands to support CID of the PDN connection.
+
+* Updated the application to use the :ref:`lib_downloader` library instead of the deprecated :ref:`lib_download_client` library.
```

Comment:
```suggestion
  * The initialization of the application to send "INIT ERROR" over to UART and show clear error log to indicate that the application is not operational in case of failing initialization.
  * The ``AT#XTCPCLI`` and ``AT#XUDPCLI`` commands to support CID of the PDN connection.
  * The application to use the :ref:`lib_downloader` library instead of the deprecated :ref:`lib_download_client` library.
```

Reply:
> 

Sorry, mistake in copy/paste, only to add one line